packages/jaeger-ui/src/components/common/ExternalLinks.tsx

```diff
@@ -39,25 +39,22 @@ const LinkValue = (props: {
 );
 
 // export for testing
-export const linkValueList = (links: Link[]) => (
-  <Menu>
-    {links.map(({ text, url }, index) => (
-      // `index` is necessary in the key because url can repeat
-      // eslint-disable-next-line react/no-array-index-key
-      <Menu.Item key={`${url}-${index}`}>
-        <LinkValue href={url}>{text}</LinkValue>
-      </Menu.Item>
-    ))}
-  </Menu>
-);
+export const linkValueList = (links: Link[]) => {
+  const menuItems: MenuProps['items'] = links.map(({ text, url }, index) => ({
+    label: <LinkValue href={url}>{text}</LinkValue>,
+    key: `${url}-${index}`,
+  }));
+
+  return [{ label: <Menu items={menuItems} />, key: 'external-links' }];
+};
 
 export default function ExternalLinks(props: ExternalLinksProps) {
```

Comment:
could you please add a comment explaining how this function is used. I can't get it from the context - are these in the top nav or elsewhere?

Reply:
After putting some dummy data, here's what it now looks like, and used to look like before:

Now:
https://github.com/jaegertracing/jaeger-ui/assets/94157520/fa66b020-4e5b-4234-8e72-d73bac5f8e8c

Before:
https://github.com/jaegertracing/jaeger-ui/assets/94157520/91af1535-1459-42a9-9fc9-e28a52f1c525

Reply:
What it looks like in latest commit:


https://github.com/jaegertracing/jaeger-ui/assets/94157520/98306751-f6d1-469e-9a5d-c8b6f9ef3723